### DISCUSSION TITLE
Which better between Morphological Tesselation or Enclosed Tessellation with street network?

### DISCUSSION THREAD

**Discussion by zhulfaliant01** [pysal/momepy#547] [Q&A] [answered]
I want to replicate the simple urban type analysis in my city. And somehow wondering which is the better between Morphological or Enclosed Tesselation to use? Or is there any cases which one is better than the other?

Thanks

**Comment by martinfleis** [comment, marked as answer]
Morphological tessellation is easier to use as there's always 1:1 correspondence between cells and buildings. Enclosed is a bit more truthful to the reality as it takes into account other boundaries but since it is a subdivision of enclosures, there are cells that do not have a building so it may sometimes require special handling and decision what to do in such cases. Enclosed is also easier to scale up.

We have developed enclosed for classifications that should cover both built and unbuilt areas, so it also depends on what you're trying to do.

**Reply by zhulfaliant01** [reply]
Thanks for the answer martin!
